lib/xwiimote.h

```diff
@@ -420,6 +420,16 @@ enum xwii_event_types {
 	 * weight-value.
 	 */
 	XWII_EVENT_BALANCE_BOARD,
+    
+    /**
+	 * Balance-Board key event
+	 *
+	 * Button event of the Balance-Board are reported via this interface
+	 * and not via the core-interface (which only reports core-buttons).
+	 * Valid button: B
```

Comment:
Typo? Quite sure this should be "Valid button: A".

Reply:
@sahoahfoa Did you have time look if the button should be A or B?  Perhaps this comment can be resolved?

Reply:
Fixed in #111.